Artsiom_Barysionak/Homework1/test1.py

```diff
@@ -0,0 +1,2 @@
+"""Printing simple string."""
```

Comment:
Usually wi use imperative mood like "Print smth". 
You shouldn't fix it it's just an tip.